autoload/ale/fixers/brittany.vim

```diff
@@ -1,13 +1,17 @@
-" Author: eborden <evan@evan-borden.com>
+" Author: eborden <evan@evan-borden.com>, ifyouseewendy <ifyouseewendy@gmail.com>, aspidiets <emarshall85@gmail.com>
 " Description: Integration of brittany with ALE.
 
 call ale#Set('haskell_brittany_executable', 'brittany')
+call ale#Set('haskell_brittany_options', '--write-mode inplace')
```

Comment:
Does brittany need this option to be set to work? If so, take this out of the options and write it into the command string below. Was this always needed, and did the brittany fixer never work, or did the behavior change in a recent version? If the behavior did change, then we probably detect which version of brittany is being run, and only add this option if needed.